CONTRIBUTING.md

```diff
@@ -41,6 +41,12 @@ tests should be engineered with this in mind:
 - Don't reuse schema/subject/topic names
 - Expect other clients to be interacting with the servers at the same time.
 
+Before running the tests make sure you have `protoc` installed. `protoc` is part of the protobuf-compiler package.
+In Fedora distributions you can install it using:
```

Comment:
Out of curiosity and based on your yesterday's talk: shouldn't we put trademark sign here for Fedora?

Reply:
And you are right, I did a quick check and it seemed we wouldn't need it. But after a more thorough search I found: https://fedoraproject.org/wiki/Legal:Trademark_guidelines#Trademark_Usage_Guidelines
Which indicates we might need it+

Reply:
Better safe than sorry in this department! :)